src/nwbinspector/nwbinspector.py

```diff
@@ -178,11 +181,11 @@ def configure_checks(
     help="Save path for the report file.",
     type=click.Path(writable=True),
 )
-@click.option("--overwrite", help="Overwrite an existing report file at the location.", is_flag=True)
 @click.option("--levels", help="Comma-separated names of InspectorMessage attributes to organize by.")
 @click.option(
     "--reverse", help="Comma-separated booleans corresponding to reversing the order for each value of 'levels'."
 )
+@click.option("--overwrite", help="Overwrite an existing report file at the location.", is_flag=True)
```

Comment:
Noticed these were out of order compared to their keyword argument ordering. Doesn't change anything since they're matched by keyword, but still an aesthetic/readability thing